app/assets/javascripts/discourse/app/helpers/route-action.js

```diff
@@ -30,13 +31,15 @@ function getRouteWithAction(router, actionName) {
 function routeAction(actionName, router, ...params) {
   assert("[ember-route-action-helper] Unable to lookup router", router);
 
-  runInDebug(() => {
-    let { handler } = getRouteWithAction(router, actionName);
-    assert(
-      `[ember-route-action-helper] Unable to find action ${actionName}`,
-      handler
-    );
-  });
+  if (!isTesting() || (isTesting() && router.currentRoute)) {
```

Comment:
I think this is the same?
```suggestion
  if (!isTesting() || router.currentRoute) {
```

Reply:
Yes I often write my conditionals like this, I think it's more explicit, but fine :+1: